pkgs/top-level/php-packages.nix

```
@@ -332,7 +332,12 @@ lib.makeScope pkgs.newScope (self: with self; {
         }
         { name = "exif"; doCheck = false; }
         { name = "ffi"; buildInputs = [ libffi ]; }
-        { name = "fileinfo"; buildInputs = [ pcre2 ]; }
+        {
+          name = "fileinfo";
+          buildInputs = [ pcre2 ];
+          patches = lib.optionals (lib.versionAtLeast php.version "8.3")
+            [ ../development/interpreters/php/fix-fileinfo-ext-php83.patch ];
```

Would be nice to include a comment with explanation and source too:


```suggestion
          patches = lib.optionals (lib.versionAtLeast php.version "8.3") [
            # Fix the extension unable to be loaded due to missing `get_module` function.
            # `ZEND_GET_MODULE` macro that creates it is conditional on `COMPILE_DL_FILEINFO` being defined.
            # https://github.com/php/php-src/issues/11408#issuecomment-1602106200
            ../development/interpreters/php/fix-fileinfo-ext-php83.patch
          ];
```

---

I'm pretty sure this will be fixed in the next alpha, but ok, I'll take care of this after lunch